tools/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_executable(
```

Comment:
tools/CMakeLists.txt should be `add_subdirectory(HogMaker)` and this code would go in tools/HogMaker/CMakeLists.txt

Reply:
That would be excessive intermediate CMakeLists.txt with only `add_subdirectory(HogMaker)`, there no point to create CMakeLists.txt on every directory level.

Reply:
I was thinking ahead of time to prepare for when we'll have more executables in `tools`, but we can keep it this way for now